tests/synthetic/test_step_function.py

```diff
@@ -30,6 +30,29 @@
             {'value': np.concatenate([np.zeros(10), np.ones(40), np.zeros(40), np.ones(10) * 20])},
             id='length_100_3_steps',
         ),
+        pytest.param(
+            {'length': 10, 'steps': [5], 'values': [(1, 2)], 'start_value': 10},
+            {'value': np.concatenate([np.tile(10, (5, 2)), np.tile((1, 2), (5, 1))])},
+            id='length_10_2_channel_single_step_with_single_start_value',
+        ),
+        pytest.param(
+            {'length': 10, 'steps': [5], 'values': [(1, 2)], 'start_value': (11, 12)},
+            {'value': np.concatenate([np.tile((11, 12), (5, 1)), np.tile((1, 2), (5, 1))])},
+            id='length_10_2_channel_single_step_with_2_channel_start_value',
+        ),
+        pytest.param(
+            {'length': 10, 'steps': [5], 'values': [(1, 2, 3, 4)], 'start_value': (11, 12, 13, 14)},
+            {'value': np.concatenate([
+                np.tile((11, 12, 13, 14), (5, 1)),
+                np.tile((1, 2, 3, 4), (5, 1)),
+            ])},
+            id='length_10_4_channel_single_step_with_start_value',
+        ),
+        pytest.param(
```

Comment:
I somehow missed to define that correctly

Reply:
wdym?

Reply:
there was a misleading test case which i copied from a single channel one (`length_100_2_channel_3_steps`) but did not adjust for 2 channels.
i decided that there's not much benefit in this, multi channel and more than two steps are already tested independently. so i removed it.